Helper/Order.php

```diff
@@ -549,17 +542,12 @@ private function deleteRedundantQuotes($quote) {
      */
     public function saveUpdateOrder($reference, $frontend = true, $bolt_trace_id = null)
     {
-
         $transaction = $this->fetchTransactionInfo($reference);
         $incrementId = $transaction->order->cart->display_id;
 
-        // load the quote from reserved order id
-        // $quote = $this->loadQuote($incrementId);
-
         // Load quote from entity id
         $quoteId = $transaction->order->cart->order_reference;
-        $quote = $this->quoteFactory->create()->load($quoteId);
-        //$quote = $this->quoteRepository->get($quoteId);
+        $quote = $this->quoteRepository->get($quoteId);
```

Comment:
Can you move it to the separate method (like "getQuoteById()")
And then we can cover it with PHPUnit test easily.

Reply:
Quote loading and saving moved to cart helper